app/views/settings/index.html.erb

```diff
@@ -26,12 +26,21 @@
 
       <%= f.field(:intellisms_username, :append => link_to(t("setting.change_sms_password"), "#")) %>
 
-      <div class="password_fields">
+      <div class="credential_fields">
         <%= f.field(:intellisms_password1, :type => :password) %>
         <%= f.field(:intellisms_password2, :type => :password) %>
       </div>
     </div>
 
+    <div class="adapter_settings" data-adapter="Twilio">
+      <%= f.field(:twilio_account_sid, :append => link_to(t("setting.change_twilio_auth_token"), "#")) %>
+
+      <div class="credential_fields">
+        <%= f.field(:twilio_auth_token1, :type => :password) %>
+        <%= f.field(:twilio_auth_token2, :type => :password) %>
```

Comment:
Not sure if we need re-entry for auth tokens. Typically I think there is just one box since everyone is copy/pasting.